### PR TITLE
feat: added action button to power on/power off cluster

### PR DESCRIPTION
**Pull request by rhartuv**
Those changes are including the necessary changes to the clusters :
1. Relocated the Actions button  
2. Added a Divider 
3. The Dropdown code is in a separate component

### CODE REVIEW

File: src/app/ClusterDetails/ClusterDetails.tsx [hunk 1]
```diff
@@ -21,14 +21,69 @@ import {
...
+    </Dropdown>
+  );
+};
+
+// export const DropdownBasic: React.FunctionComponent = () => { 
+//   return (
+//   <Button> 
+//     roni
+//   </Button>)
+// }
```
Comment (ikrispin) [inline src/app/ClusterDetails/ClusterDetails.tsx:83]: Maybe we can delete these lines?

**Comment by r2dedios** [comment]
Does this code enable or disable the Actions dropdown menu options based on the Cluster Status?
In other words, if the cluster is Stopped, Is the "Power Off" option available?

**Comment by vbelouso** [comment]
@rhartuv @ikrispin 

1. Please install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode to VSCode. It will format your code to a more consistent style.
2. You don't need an additional `DividerUsingDiv` component, use vanilla `<Divider />`. Place it one line above the code
```typescript
<PageSection type="tabs" variant={PageSectionVariants.light} isWidthLimited>
```
3. This code part is also not needed  because you don't use it in the code
```typescript
const [cluster, setClusterData] = useState<ClusterData>({
    count: 0,
    clusters: []
  });
```

**Review by vbelouso — CHANGES_REQUESTED** [review]
@rhartuv @ikrispin 

1. Please install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode to VSCode. It will format your code to a more consistent style.
2. You don't need an additional `DividerUsingDiv` component, use vanilla <Divider />. Place it one line above the code `<PageSection type="tabs" variant={PageSectionVariants.light} isWidthLimited>`
3. This code part is also not needed because you don't use it in the code
```typescript
const [cluster, setClusterData] = useState<ClusterData>({
    count: 0,
    clusters: []
  });
```

**Comment by vbelouso** [comment]
Good job, thanks @rhartuv @ikrispin

**Comment by r2dedios** [comment]
Closes #21

**Pull request merged**